data/migrations/V0067__filter_mur_view.sql

```diff
@@ -0,0 +1,32 @@
+/*
+Issue #2993
+PR #3010
+Filter existing view cases_with_parsed_case_serial_numbers
+    by case_type = 'MUR'
+*/
+
+SET search_path = fecmur, pg_catalog;
+
+DROP VIEW IF EXISTS cases_with_parsed_case_serial_numbers_tmp;
+
+CREATE VIEW cases_with_parsed_case_serial_numbers_tmp AS
```

Comment:
We don't need to create a view with an `_tmp` suffix and then rename it at the end. PostgreSQL provides `CREATE OR REPLACE VIEW...` to do this in one shot - https://www.postgresql.org/docs/10/static/sql-createview.html